forge/src/main/java/org/openjdk/skara/forge/github/GitHubRepository.java

```diff
@@ -261,14 +261,23 @@ public VCS repositoryType() {
     }
 
     @Override
-    public String fileContents(String filename, String ref) {
-        var conf = request.get("contents/" + filename)
-                          .param("ref", ref)
-                          .execute().asObject();
+    public Optional<String> fileContents(String filename, String ref) {
+        JSONValue conf;
```

Comment:
That's a bit odd name for the variable. I realize you did not come up with it, but I think it is a remnant from the time this function was only used to read the configuration file. I recommend changing it to `content`.